### DISCUSSION TITLE
Portainer Business Edition

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#124] [Ideas]
[Portainer extension](https://hub.docker.com/extensions/portainer/portainer-docker-extension) already exists, but this request is to add business edition capabilities to the existing extension or create a new one to support those features.

Note: This is a transfer from our initial method of collecting new extension requests. If you have additional information on how this will help you or other functionality you'd like to see, please add more details in the comments.

**Comment by friendsfan** [comment]
Can we please habve that? Portainer now gives away BE licenses (limited to cetain number of nodes) for all users, so really no need to stick to the CE.

https://www.portainer.io/take-5
